### ISSUE TITLE
lp_ticker_get_overflows_counter - should be implemented

### ISSUE THREAD

**Issue by 0xc0170** [ARMmbed/mbed-hal-nuc472#1]
```
uint32_t lp_ticker_get_overflows_counter(void)
{
    // FIXME: Not know what overflow means here
    return 0;
}
```

Based on the comment above, if the doxygen comments in the function declaration are not clear, please create an issue, we can review the docs.

To add some details, lp ticker has a counter 32bit (most of the MCU have at least one timer with counter in range <16,32) bits. The overflow counter helps to provide additional 32bit counter to form 64 bit timestamps.

@ccli8

**Comment by ccli8** [comment]
@0xc0170 I implemented it in V1.1.20.

**Comment by 0xc0170** [comment]
Note: I'll send PR to add more details to sleep_until and set_interrupt for lp ticker , to clarify what is the intention with `now` and `until`.
